fal_scripts/fal_github_insights.py

```diff
@@ -0,0 +1,38 @@
+import os
+import pycurl
+import json
+import certifi
+from io import BytesIO
+import pandas as pd
+
+ACCESS_TOKEN = os.environ['GITHUB_TOKEN']
+
+g_url = 'https://api.github.com/repos/fal-ai/fal/traffic/'
+header_list = [f'Authorization: token {ACCESS_TOKEN}']
+metrics = {'clones': '', 'views': '', 'popular/paths': '', 'popular/referrers': ''}
+
+for metric in metrics.keys():
+    buffer = BytesIO()
```

Comment:
I think you could use the `requests` package here to make this easier

Reply:
Ok, will check it.